setup.py

```diff
@@ -23,7 +23,8 @@
     install_requires=[
         'futures',
         'protobuf>=3.0.0b2.post2',
-        'opentracing>=1.2.1,<1.3'
+        'opentracing>=1.2.1,<1.3',
+        'six<2.0',
```

Comment:
Well, 1.10.0 is the recommended version these days as it's the newest - almost two years old. That being said, 1.9 is pretty popular (two years old). Some packages even simply require the six module (i.e. no version required). 

Changelog/releases don't provide much information regarding breaking changes over versions. Suggestions here?